runner/ansible/vars/azure_env.yml

```diff
@@ -4,3 +4,11 @@ expected:
   "1.1.1": "30000"
   "1.2.1": "true"
   "1.3.5": "60"
+  "1.1.2": "36000"
```

Comment:
It think we should have the `ids` in numerical order, to make the reading easier, like it is in the `dev` vars file (I mean to put `1.1.2` before `1.2.1`

Reply:
agreed and I did mean to do that. thanks.